hack/local-development/start-gardenlet

```diff
@@ -56,6 +56,50 @@ else
   esac
 fi
 
+# generate seed-specific client certificate for gardenlet
```

Comment:
Does it make sense to move this block further to the top (can probably also be extracted to a function) and remove `local-garden/kubeconfigs/gardenlet.conf` and `local-garden/certificates/certs/gardenlet.crt`?